aws-ami.pkr.hcl

```diff
@@ -40,25 +46,63 @@ variable "build_subnet_id" {
   type        = string
 }
 
+variable "source_ami_name_filter_arm64" {
+  description = "Name filter for the source arm64 AMI image"
+  type        = string
+}
+
+variable "source_ami_name_filter_x86-64" {
+  description = "Name filter for the source x86-64 AMI image"
+  type        = string
+}
+
 # Defines a local variable to generate a timestamp for AMI naming.
 locals {
   timestamp = regex_replace(timestamp(), "[- TZ:]", "")
 }
 
 # Defines the source configuration for the Amazon EBS builder.
-source "amazon-ebs" "ubuntu" {
-  ami_name      = "ami-ontotext-graphdb-${var.gdb_version}-${local.timestamp}"
-  instance_type = "${var.build_instance_type}"
+source "amazon-ebs" "ubuntu-x86-64" {
+  ami_name      = "ami-ontotext-graphdb-x86-64-${var.gdb_version}-${local.timestamp}"
+  instance_type = "${var.build_instance_type_x86-64}"
+  vpc_id        = "${var.build_vpc_id}"
+  subnet_id     = "${var.build_subnet_id}"
+  ami_regions   = "${var.build_aws_regions}"
+
+
+  # Specify the source AMI to use as a base.
+  source_ami_filter {
+    filters = {
+      name                = "${var.source_ami_name_filter_x86-64}"
+      root-device-type    = "ebs"
+      virtualization-type = "hvm"
+
+    }
+    most_recent = true
+    owners      = ["099720109477"] # AWS account ID of the owner.
+  }
+
+  # SSH configuration for connecting to the instance.
+  ssh_username                = "ubuntu"
+  associate_public_ip_address = true
+  ssh_interface               = "public_ip"
+}
+
+source "amazon-ebs" "ubuntu-arm64" {
+  ami_name      = "ami-ontotext-graphdb-arm64-${var.gdb_version}-${local.timestamp}"
+  instance_type = "${var.build_instance_type_arm64}"
   vpc_id        = "${var.build_vpc_id}"
   subnet_id     = "${var.build_subnet_id}"
   ami_regions   = "${var.build_aws_regions}"
 
+
```

Comment:
I don't get the extra empty lines..

Reply:
[81](https://github.com/Ontotext-AD/packer-aws-graphdb/pull/3/files#diff-c52a56b091dcb1b64003a082b0250bcffa5ff3760581337a85b5e53e982a9cf7R81) & [106](https://github.com/Ontotext-AD/packer-aws-graphdb/pull/3/files#diff-c52a56b091dcb1b64003a082b0250bcffa5ff3760581337a85b5e53e982a9cf7R106)

Reply:
Fixed